src/components/bills/BillNavigationTab.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+	type TabId = 'status' | 'proposer' | 'party';
+
+	interface Tab {
+		id: TabId;
+		label: string;
+		disabled?: boolean;
+	}
+
+	export let tabs: Tab[];
+
+	let activeTab: TabId;
+
+	$: {
+		if (tabs.length) {
+			activeTab = tabs[0].id;
+		}
+	}
+
+	const onClickTab = (tab: TabId) => {
+		document.getElementById(tab)?.scrollIntoView({
+			behavior: 'smooth'
+		});
+		activeTab = tab;
+	};
+</script>
+
+<div class="flex w-full">
+	{#each tabs as { id, label, disabled } (id)}
+		<button
+			class={activeTab === id ? 'tab-active' : 'tab-inactive'}
+			on:click={() => onClickTab(id)}
+			{disabled}
+			style="text-align:center"
```

Comment:
Look like `text-align:center` and Tailwind `text-left` in the style tag is conflict? I think you can remove the style and replace `text-left` with `text-center` or there is a reason behind this?